StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1615UnitTests.cs

```diff
@@ -106,14 +133,106 @@ public class ClassName
     /// </summary>
 $$
 }";
-            var expected = new[]
-            {
-                this.CSharpDiagnostic().WithLocation(10, 21)
-            };
+            var fixedCode = @"
+/// <summary>
+/// Foo
+/// </summary>
+public class ClassName
+{
+    /// <summary>
+    /// Foo
+    /// </summary>
+    /// <returns></returns>
+$$
+}";
+
+            var expected = this.CSharpDiagnostic().WithLocation(10, 21);
 
             await this.VerifyCSharpDiagnosticAsync(testCode.Replace("$$", declaration), expected, CancellationToken.None).ConfigureAwait(false);
+            await this.VerifyCSharpDiagnosticAsync(fixedCode.Replace("$$", declaration), EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+            await this.VerifyCSharpFixAsync(testCode.Replace("$$", declaration), fixedCode.Replace("$$", declaration), cancellationToken: CancellationToken.None).ConfigureAwait(false);
         }
-        
+
+        [Theory]
+        [MemberData(nameof(AsynchronousWithReturnValue))]
+        public async Task TestAsynchronousMethodWithReturnTypeWithoutReturnTypeDocumentationAsync(string declaration)
+        {
+            var testCode = @"
+using System.Threading.Tasks;
+/// <summary>
+/// Foo
+/// </summary>
+public class ClassName
+{
+    /// <summary>
+    /// Foo
+    /// </summary>
+$$
+}";
+            var fixedCode = @"
+using System.Threading.Tasks;
+/// <summary>
+/// Foo
+/// </summary>
+public class ClassName
+{
+    /// <summary>
+    /// Foo
+    /// </summary>
+    /// <returns><placeholder>A <see cref=""Task""/> representing the asynchronous operation.</placeholder></returns>
```

Comment:
The code fix for #606 generates content for a `<value>` documentation element according to the current content of the `<summary>` element. It also supports the `BatchFixAllProvider` to quickly create `<value>` documentation for all properties that already have `<summary>` documentation. However, this documentation is only a guess of what the user intended, so we wrap this content in a `<placeholder>` element which indicates that the content still needs to be reviewed for accuracy. After reviewing the content, the `<placeholder>` element is removed leaving just its content (the code fix in #604 can do this automatically).

The `<placeholder>` element is not intended to affect the rendered documentation. Its entire purpose is to mark sections of documentation as "needing review", so we use it any time content is generated for a comment by an analyzer.